docs/develop/getting-started/virtualenvs.md

```diff
@@ -24,7 +24,7 @@ programs (i.e. a distribution provides the ``python`` command).
 ### Distribution version
 
 Each distribution comes in multiple versions that's able to interpret the
-Python programming language - e.g Python 3.8, 3.9 etc.
+Python programming language - e.g., Python 3.8 and 3.9
```

Comment:
`Each distribution supports multiple version of the Python programming language e.g., Python 3.8 and Python 3.9`

(can't create a suggestion for unmodified line, but let's reword it too)